include/deal.II/fe/fe_interface_values.h

```diff
@@ -89,6 +89,45 @@ namespace FEInterfaceViews
     using third_derivative_type =
       typename FEValuesViews::Scalar<dim, spacedim>::third_derivative_type;
 
+    /**
+     * An alias for the data type of the product of a @p Number and the
+     * values of the view this class provides. This is the data type of
+     * vector components of a finite element field whose degrees of
+     * freedom are described by a vector with elements of type @p Number.
+     */
+    template <typename Number>
+    using solution_value_type = typename ProductType<Number, value_type>::type;
+
+    /**
+     * An alias for the data type of the product of a @p Number and the
+     * gradients of the view this class provides. This is the data type of
+     * scalar components of a finite element field whose degrees of
+     * freedom are described by a vector with elements of type @p Number.
+     */
+    template <typename Number>
+    using solution_gradient_type =
+      typename ProductType<Number, gradient_type>::type;
+
+    /**
+     * An alias for the data type of the product of a @p Number and the
+     * hessians of the view this class provides. This is the data type of
```

Comment:
```suggestion
     * Hessians of the view this class provides. This is the data type of
```

Reply:
(I know that this is wrong in the docs for the normal `FEValuesViews`. We should fix that too, but in another PR.)